src/hooks/disputable-voting-logic.js

```diff
@@ -10,6 +11,50 @@ function useGetVotes() {
   return votes
 }
 
+export function useGetVote(voteId) {
+  const [loading, setLoading] = useState(false)
+  const [extendedProperties, setExtendedProperties] = useState(null)
+  const [disputableVoting] = useApp('disputable-voting')
+  const [vote] = useDisputableVoting(disputableVoting, (app) => {
+    return app.vote(voteId)
```

Comment:
[It would throw](https://github.com/aragon/connect/blob/ef0f35b82bd1a6c842a75cc2ff936a971e11185d/packages/connect-voting-disputable/src/thegraph/parsers/votes.ts#L58), we should try / catch this.